larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -87,6 +87,54 @@ namespace lar_pandora {
         else if (!isDualPhase)
           listOfGaps.emplace_back(LArDetectorGap(X1, Y1, Z1, X2, Y2, Z2));
       }
+      if (isDualPhase)
+      {
+        for (LArDaughterDriftVolumeList::const_iterator iterDghtr1 = driftVolume1.GetTpcVolumeList().begin(),
+                 iterDghtrEnd1 = driftVolume1.GetTpcVolumeList().end();
+             iterDghtr1 != iterDghtrEnd1; 
+             ++iterDghtr1)
```

Comment:
Would have these on one line. Also only one "tab" of indentation for the line break in the for loop specification.